docker-storage-setup.sh

```diff
@@ -978,7 +976,7 @@ setup_extra_volume() {
 }
 
 setup_extra_lv_fs() {
-  [ -z "$CONTAINER_ROOT_LV_MOUNT_PATH" ] && [ -z "$CONTAINER_ROOT_LV_NAME" ] && return 0
+  [ -z "$RESOLVED_MOUNT_DIR_PATH" ] && [ -z "$CONTAINER_ROOT_LV_NAME" ] && return 0
```

Comment:
You are buying that someone later coming and reading this code will know what is going on.
The very next line uses RESOLVED_MOUNT_DIR_PATH.  No where in the function is the CONTAINER_ROOT_LV_MOUNT_PATH used.